package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "hydraclient.js",
+    "version": "0.0.1",
+    "scripts": {
+        "test": "node node_modules/karma/bin/karma start",
+        "test-once": "node node_modules/karma/bin/karma start --single-run"
```

Comment:
I have no opinion here. Whichever is better from a developer point of view (not a tool point of view)